HW7/HW7.cpp

```diff
@@ -0,0 +1,70 @@
+#include <iostream>
+#include <cstdlib>
+#include <ctime>
+
+void guessNum() {
+    std::cout << "This is the Number Guessing Game!\nThis game generates random number (0 - 50) and you must guess it! \nIf you want to exit just enter -1 \n\n";
+    int num = 0;
+    int bestRound = 0;
+   while(true)
+   {
+       if (num == -1) {
+           std::cout << "Huh! Looser! Come back when you'll be brave enough!\n";
+           break;
+       }
+       int rounds = 1;
+       std::srand(std::time(nullptr));
```

Comment:
srand should be called only ONCE per launch. You may observe strange artifacts in randomizer if you analyze your rand numbers well enough calling it that often